### PR TITLE
Remove unused functions getPing and resetPing

### PR DESCRIPTION
**Pull request by agarciamontoro**
#### Summary
I was looking for something else and stumbled upon this dead code that was polluting my searches, so I wondered I could just remove it :)

Super low priority, of course.

#### Ticket Link
--

#### Screenshots
--

#### Release Note
```release-note
NONE
```

### CODE REVIEW

**Review by M-ZubairAhmed — APPROVED** [review]

File: webapp/channels/src/packages/mattermost-redux/src/actions/general.ts [hunk 1]
```diff
@@ -16,39 +16,6 @@ import {logError} from './errors';
...
-            return {error: pingError};
-        }
-
-        return {data};
-    };
-}
-
-export function resetPing(): ActionFunc {
-    return async (dispatch: DispatchFunc) => {
-        dispatch({type: GeneralTypes.PING_RESET, data: {}});
```
[the remark this answers is not in the document]
Comment (M-ZubairAhmed) [reply webapp/channels/src/packages/mattermost-redux/src/actions/general.ts:46]: yes

**Review by M-ZubairAhmed — APPROVED** [review]

**Review by harshilsharma63 — APPROVED** [review]
Changes look fine but I think there are some unused imports due to removed code.

**Comment by agarciamontoro** [comment]
> Changes look fine but I think there are some unused imports due to removed code.

Whoops. Should be fixed now, let's see if CI is happy now :)

**Pull request merged**